internal/rbd/controllerserver.go

```diff
@@ -226,6 +221,7 @@ func validateRequestedVolumeSize(rbdVol, parentVol *rbdVolume, rbdSnap *rbdSnaps
 }
 
 // CreateVolume creates the volume in backend.
+// nolint:gocyclo // encrypted cloning/snapshots added more complexity, this needs to be addressed
```

Comment:
fwiw, I have an extra commit that cleans up the `nolint` annotation. It will be posted with the next update of this PR.